modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -1278,17 +1286,17 @@ public static string PadDecimals(this string instance, int digits)
         public static string PadZeros(this string instance, int digits)
         {
             string s = instance;
-            int end = s.Find(".");
+            int end = s.IndexOf('.');
 
-            if (end == -1)
+            if (end < 0)
```

Comment:
The C++ implementation seems to use [`== -1`](https://github.com/godotengine/godot/blob/9dd6c4dbac70b28e8156255c3a2b78722772b036/core/string/ustring.cpp#L4669) here.